tests/test_airdrops.py

```diff
@@ -42,8 +42,11 @@ def test_files_and_jsons():
             with open(airdrop['file_path'], 'br') as f:
                 assert airdrop['file_hash'] == hashlib.sha256(f.read()).hexdigest(), f'Invalid hash for {airdrop["file_path"]}'
             df = polars.read_parquet(airdrop['file_path'])
-            for i, (address,) in enumerate(df.select(polars.selectors.by_index(0)).rows()):
-               assert is_checksum_address(address), f'{airdrop["file_path"]} address {address} is not checksummed at row {i+2}'
+            assert df.columns[:2] == ['address', 'amount'], f'{airdrop["file_path"]} does not have address and amount columns'
```

Comment:
what happens for files that have a different header?

Reply:
currently they all have the header ['address', 'amount'], and it's also documented that any new airdrop must have those headers
to remind/ensure the structure